src/en/proposals/emogarbage-celestial-war-gamemode.md

```diff
@@ -0,0 +1,327 @@
+- **Solar Gauntlets:** Lightly armored gloves that, when worn, cause all melee attacks to ignite the person hit.
+- **Cloak of Embers:** A cape with extremely high temperature and heat damage resistance.
+- **Hellfire Stone:** A small explosive: when detonated, explodes into a ring of fireballs, which severely burn and ignite targets. 
+
+They have the following abilities:
+- **Living Flames:** Spawns 3 fire spirits in a target location.
+These are friendly towards Khull's followers but will attack everyone else, setting them on fire every hit. 
+They can be countered by splashing water on them.
+- **Fireball:** A fireball descends on a target location, creating a medium-sized explosion on landing.
+- **Crater:** An upgraded version of Fireball; an even larger fireball descends, creating a larger explosion and coating the ground in lava and basalt.
+
+### Helion of Mercury
+_An impossibly large man dressed in a pinstripe suit. His skin is pure gold: it shines with eternal malice._
+
+Helion lays domain over wealth and greed.
+Their divinity is raised through the purchase of goods from Cargo.
+More expensive items scale higher in value as well as more unique purchases, rather than the same one over and over.
+They additionally accept sacrifices of valuables, which give divinity based on the price of the item sacrificed.
+
+They have the following artifacts:
+- **Celestial Scale:** A handheld scale that does varying amounts of damage based on the price of the person wearing it.
+- **Golden Suit:** A suit weaved of pure gold. 
+Reflects projectiles and lasers when not covered by outerclothing.
+- **Gilded Statuette:** A golden statue of Helion. 
+Sells at Cargo for a high price. 
+
+They have the following abilities:
+- **Blockade:** Creates a line of strong golden statues of Helion, blocking a path.
+- **Midas Touch:** All walls in a small radius are turned to gold before melting, clearing open a pathway for disciples.
+- **Golden Domain:** A large radius around Helion is converted to pure gold.
+All non-followers in the radius are temporarily transformed into invulnerable golden statues, reverting to normal after a minute.
+
+### Aquila of Venus
+_A humanoid form engulfed in rapid lightning. The electricity forms a spiky crown around their head._
+
+Aquila controls and is powered by electricity.
+Followers of Aquila are capable of creating **Celestial Conduits**, which, when powered by HV cables, produce a small amount of Divinity every second while powered.
+Aquila also gains power from when crew members are shocked or electrocuted. 
+Charged objects and batteries can additionally be sacrificed for low amounts of divinity based on the current charge.
+
+They have the following artifacts:
+- **Throwing Bolt:** A lightning bolt able to be held in hand.
+When thrown, deals shock damage on collision and electrocutes whoever it hit.
+- **Faraday Helm:** A moderately strong golden helmet that gives protection from electrocution.
```

Comment:
Insul helmet? Feels underpowered for a god who will most likely build up a following in engi

Reply:
Hell, even if this prevents 100% of shock damage, shock damage isn't used enough, I think this artifact needs changing